api/tests/test_main.py

```diff
@@ -0,0 +1,342 @@
+import os
+import refget
+import logging
+import hashlib
+import pytest
+from importlib import reload
+
+os.environ["INDEXDBPATH"] = "./testdata/indexdb.tkh"
+os.environ["SEQPATH"] = "./testdata/"
```

Comment:
Wouldn't it be consistent to read this from env as you are doing for the main here ?
https://github.com/Ensembl/ensembl-refget/blob/f49cf77b64da7ccf04c3c106fb7636cd1158428f/api/src/refget/main.py#L66

Reply:
Well, this is the test and I want to set these to this value.
I could us the .env file, but for the test, I feel it makes no difference.

The main is written so that it grabs these valuea from either the env or the .env file.

Arguably, I could test that setting it from the .env file works. Noted for the next round of improvements.